File: abc159/c/c/c.c
#include <stdio.h>

double vmax(double wa);

int main(void) {
  int l;
  scanf("%d", &l);
  printf("%lf\n", vmax(l));

  return 0;
}

double vmax(double wa) { return wa / 3.0 * wa / 3.0 * wa / 3.0; }

File: abc159/c/c.rs
use std::io::Read;

fn vmax(x: f64) -> f64 {
    x * x * x
}

fn main(){
    let mut buf = String::new();
    std::io::stdin().read_to_string(&mut buf).unwrap();
    
    let l: f64 = buf.split_whitespace().next().unwrap().parse().unwrap();
    let x = l / 3.0;
    println!("{:.12}", vmax(x));
}
